src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -82,6 +84,7 @@ def test_dmg_storage_query_devices(self):
         if self.targets != targets:
             self.fail("Wrong number of targets found: {}".format(targets))
 
+    @skipForTicket("DAOS-4286")
```

Comment:
DAOS-4286 is **Resolved**.  You can't skip a test for a ticket that has been resolved.